File: src/com/dengzii/plugin/adb/ui/AboutDialog.java
package com.dengzii.plugin.adb.ui;

import com.intellij.ide.BrowserUtil;
import com.intellij.ide.plugins.PluginManager;
import com.intellij.openapi.extensions.PluginId;

import javax.swing.*;
import java.awt.*;
import java.awt.event.MouseAdapter;
import java.awt.event.MouseEvent;
import java.util.Arrays;

public class AboutDialog extends JDialog {
    private JLabel labelVersion;
    private JPanel content;
    private JLabel reference;
    private JLabel rating;
    private JLabel sourceCode;
    private JLabel issue;
    private JLabel labelFree;

    public static void show_() {
        Dialog dialog = new AboutDialog();
        dialog.pack();
        dialog.setVisible(true);
    }

    AboutDialog() {
        setContentPane(content);
        setModal(true);

        try {
            labelVersion.setText(Arrays.stream(PluginManager.getPlugins()).filter(ideaPluginDescriptor -> {
                return ideaPluginDescriptor.getPluginId() == PluginId.getId("com.dengzii.plugin.adb");
            }).findFirst().get().getVersion());
        } catch (Throwable ignored) {

        }

        setUrlEvent(reference, "https://developer.android.com/studio/command-line/adb");
        setUrlEvent(rating, "https://plugins.jetbrains.com/plugin/13156-android-wifiadb");
        setUrlEvent(sourceCode, "https://github.com/dengzii/WiFiADB");
        setUrlEvent(issue, "https://github.com/dengzii/WiFiADB/issues");
        setUrlEvent(labelFree, "https://github.com/dengzii/WiFiADB/releases");
    }

    private void setUrlEvent(JLabel label, String url) {
        label.setCursor(Cursor.getPredefinedCursor(Cursor.HAND_CURSOR));
        label.addMouseListener(new MouseAdapter() {
            @Override
            public void mousePressed(MouseEvent e) {
                super.mousePressed(e);
                BrowserUtil.browse(url);
            }
        });
    }

    @Override
    public void pack() {
        super.pack();
        Dimension screen = Toolkit.getDefaultToolkit().getScreenSize();
        int w = getWidth();
        int h = getHeight();
        int x = screen.width / 2 - w / 2;
        int y = screen.height / 2 - h;
        setLocation(x, y);
        setPreferredSize(new Dimension(w, h));
    }
}

File: src/com/dengzii/plugin/adb/ui/AboutDialog.kt
package com.dengzii.plugin.adb.ui

import com.dengzii.plugin.adb.tools.ui.XDialog
import com.intellij.ide.BrowserUtil
import com.intellij.ide.plugins.PluginManager
import com.intellij.openapi.extensions.PluginId
import com.intellij.ui.layout.applyToComponent
import com.intellij.ui.layout.panel
import com.intellij.util.ui.UIUtil
import java.awt.Dimension
import java.awt.Font

class AboutDialog : XDialog("About") {

    private val version by lazy {
        PluginManager.getPlugins().first {
            it.pluginId == PluginId.getId("com.dengzii.plugin.adb")
        }.version
    }

    init {
        persistDialogState = false
        contentPane = panel {
            row {
                label("")
            }
            row {
                label("WiFiADB", style = UIUtil.ComponentStyle.LARGE, bold = true).applyToComponent {
                    font = UIUtil.getLabelFont().deriveFont(24f).deriveFont(Font.BOLD)
                }.withLargeLeftGap()
            }
            row {
                cell {
                    label("Version: ").withLargeLeftGap()
                    label(version)
                }
            }
            row {
                cell {
                    label("Reference:").withLargeLeftGap()
                    link("Android Debug Bridge") {
                        browser("https://developer.android.com/studio/command-line/adb")
                    }
                }
            }
            row {
                cell {
                    label("Rating:").withLargeLeftGap()
                    link("JetBrains Plugin Repository") {
                        browser("https://plugins.jetbrains.com/plugin/13156-android-wifiadb")
                    }
                }
            }
            row {
                cell {
                    label("Source Code:").withLargeLeftGap()
                    link("https://github.com/dengzii/WiFiADB") {
                        browser("https://github.com/dengzii/WiFiADB")
                    }
                }
            }
            row {
                cell {
                    label("If you have any questions or suggestions, please").apply {
                        withLargeLeftGap()
                    }.applyToComponent {
                        autoscrolls = true
                    }
                    link("Create Issue  ") {
                        browser("https://github.com/dengzii/WiFiADB/issues/new")
                    }
                }
            }
            row {
                label("")
            }
            row {
                label("")
            }
            row {
                label("Having a good day!",
                        fontColor = UIUtil.FontColor.BRIGHTER,
                        style = UIUtil.ComponentStyle.SMALL)
                        .withLargeLeftGap()
            }
        }
    }

    override fun onOpened() {
        super.onOpened()
        size = Dimension(400, 300)
        location = getLocationCenterOfScreen()
    }

    private fun browser(url: String) {
        BrowserUtil.browse(url)
    }
}